mrbgems/mruby-eval/test/eval.rb

```
@@ -65,6 +65,25 @@
   end
 end
 
+assert 'eval a string literal at the width of the pool length' do
+  # The dump records a pool string's length in 16 bits, so 65535 bytes is the
+  # longest one that survives the round trip into an mrb_irep.
+  assert_equal 65535, eval('"' + 'x' * 65535 + '"').size
+  assert_raise(SyntaxError) do
+    eval('"' + 'x' * 65536 + '"')
```

RuboCop (1.88.2)

[convention] The use of `eval` is a serious security risk.

(Security/Eval)